oauth2/src/main/java/com/okta/spring/oauth/ConfigurableAccessTokenConverter.java

```diff
@@ -74,7 +77,16 @@ public OAuth2AccessToken extractAccessToken(String value, Map<String, ?> map) {
 
     @Override
     public OAuth2Authentication extractAuthentication(Map<String, ?> map) {
```

Comment:
Can you name "map" in a better way that suits this context?

Reply:
This is the same name as the super class